### PR TITLE
fix(ingest): hide main github readme from docs site

### PR DESCRIPTION
**Pull request by hsheth2**
Will fix search and some SEO issues.

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/adding-source.md [hunk 1]
```diff
@@ -130,8 +130,8 @@ class FileSource(Source):
 
 #### 7.2 Write custom documentation
 
-- Create a copy of [`source-docs-template.md`](./source-docs-template.md) and edit all relevant components. 
-- Name the document as `<plugin.md>` and move it to `metadata-ingestion/docs/sources/<platform>/<plugin>.md`. For example for the Kafka platform, under the `kafka` plugin, move the document to `metadata-ingestion/docs/sources/kafka/kafka.md`.
+- Use the docstring section of the source class to add documentation. This section supports markdown formatting.
```
Comment (maggiehays) [inline metadata-ingestion/adding-source.md:133]: Does the docstring section contain all of the relevant detail that was available in `source-docs-template.md` ?

**Review by maggiehays — APPROVED** [review]

**Comment by shirshanka** [comment]
@hsheth2 : is this still an active PR?

**Comment by hsheth2** [comment]
Yep - just updated it, and will merge it when CI is green.

**Comment by hsheth2** [comment]
We'll need to think about what to do with our README in a bit more detail.

**Pull request closed without merging**